### PR TITLE
Update docs for offline migration errors / Card 4616

### PR DESCRIPTION
**Pull request by wstephenson**
And indicate that target_base_product is mandatory

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/SCC-Connect-Prophet/74396/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged c43f31fa53edfe82fb2a1ef71281094ccd1ea514 into 154d8e17cd0c384304ac26119d89bf224dccb945 )

**Comment by suse-tests-fail** [comment]
Prophet reports failure.
https://ci.suse.de/job/SCC-Connect-Prophet/74417/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
Failed fast in project connect on step docker-run-tests.
( Failure: Merged c43f31fa53edfe82fb2a1ef71281094ccd1ea514 into cbea5ee77d7f957bb6d087858b8f99cf59eb5b03 )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/SCC-Connect-Prophet/76135/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged c43f31fa53edfe82fb2a1ef71281094ccd1ea514 into cbea5ee77d7f957bb6d087858b8f99cf59eb5b03 )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/SCC-Connect-Prophet/76411/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged c43f31fa53edfe82fb2a1ef71281094ccd1ea514 into 61218fdb538dddfa37dca7b4de4f16cd56fd36c2 )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/SCC-Connect-Prophet/76412/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged c43f31fa53edfe82fb2a1ef71281094ccd1ea514 into ba96cc42ec5a7da20f8ac83d95eed926e0ee858e )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/SCC-Connect-Prophet/77850/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged 62f4806792d1cbc84f7e4ef47d5aad065369072a into ba96cc42ec5a7da20f8ac83d95eed926e0ee858e )

**Comment by suse-tests-pass** [comment]
@thutterer will review your pull request :dancers: check https://trello.com/c/MTT0BvSi/4616-8-p1sles15-migration-support-online-and-offline-migration-targets
  @thutterer: Please review this pull request using our guidelines:
  * test for acceptance criteria / functionality
  * check if the new code is covered with tests
  * check for unintended consequences
  * encourage usage of the boyscout rule
  * make sure the code is architected in the best way
  * check that no unnecessary technical debt got introduced
  * make sure that no unnecessary FIXMEs or TODOs got added

File: doc/SCC-API-(Implemented).md [hunk 1]
```diff
@@ -1177,6 +1177,15 @@ Status: 200 OK
...
+- 422: `The requested product '%s' is not activated on this system.`
+- 422: `Required parameters are missing or empty: installed_products`
+- 500: `No base product found` - When the `installed_products` parameter contains no base product.
+- 500: `Multiple base products found: ["SUSE Linux Enterprise Server 12 SP4 x86_64", "SUSE Linux Enterprise Server 12 SP3 x86_64"]` -
+When the `installed_products` parameter contains more than one base product.
+This should not happen, since a system can only have one base product installed.
+
 ##### List System Offline Migrations
 
 Given a list of installed products, return all possible online migration paths.
```
Comment (thutterer) [inline doc/SCC-API-(Implemented).md]: It should say "return all possible offline migration paths." here, right?
Can you maybe fix that in this PR, @lagartoflojo

Comment (lagartoflojo) [reply doc/SCC-API-(Implemented).md]: Nice catch, thanks!

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/SCC-Connect-Prophet/78083/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged cd8855b05e9d70035a73d4ef56c5861e4972443f into ba96cc42ec5a7da20f8ac83d95eed926e0ee858e )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/SCC-Connect-Prophet/78085/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged 0c7e7c9ca2e43ebce08c33413062d0bc28fb4e5a into ba96cc42ec5a7da20f8ac83d95eed926e0ee858e )

**Pull request merged**